src/Agent.Worker/TestResults/FileZipper.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+
+namespace Microsoft.VisualStudio.Services.Agent.Worker.TestResults
+{
+    internal class FileZipper : IZipper
```

Comment:
Why is file zipper under the test results code.  Seems like it should be in common utilities and made generic